test/run.test.js

```diff
@@ -366,7 +368,7 @@ test('run should accept a unix socket/windows pipe', (t) => {
 })
 
 for (let i = 1; i <= 5; i++) {
-  test(`run should count all ${i}xx status codes`, (t) => {
+  test(`run should count all ${i}xx status codes`, { skip: isNode15 }, (t) => {
```

Comment:
```suggestion
  // TODO we should not skip those tests
  test(`run should count all ${i}xx status codes`, { skip: isNode15 }, (t) => {
```

Reply:
Sure, I'm actually gonna look into the issue right after 🙂